modules/upgrade/pages/db-migration.adoc

```diff
@@ -1,61 +1,65 @@
+Direct migration from PostgreSQL 9.4 to version 10 is not supported.
 
-{productname}{nbsp}3 uses PostgreSQL{nbsp}9.4 by default.
-{sls}{nbsp}12{nbsp}SP3 uses PostgreSQL{nbsp}9.6 by default.
-
-This chapter provides information on managing your PostgreSQL database, and provides instructions for upgrading your database to PostgreSQL{nbsp}9.6.
-
-{productname}{nbsp}3.1 on {sls}{nbsp}12{nbsp}SP3 and {productname}{nbsp}3.2 use PostgreSQL{nbsp}9.6 by default.
-If you use one of those you can skip this section.
 
 
 
+[[db-migration-prepare]]
 == Prepare to Upgrade
 
-Before you begin the upgrade, prepare your existing system and create a database backup.
+Before you begin the upgrade, prepare your existing 3.2 system and create a database backup.
 
 PostgreSQL stores data at [path]``/var/lib/pgsql/data/``, and logs to [path]``/var/lib/pgsql/data/pg_xlog/``.
 
-
 .Procedure: Preparing to Upgrade
+
 . Check the active PostgreSQL version:
 +
 ----
 psql --version
 ----
 +
-If you are using PostgreSQL{nbsp}9.4, you can upgrade to PostgreSQL{nbsp}9.6.
+If you are using PostgreSQL{nbsp}9.6, you can upgrade to PostgreSQL{nbsp}10.
+
 . Check the active smdba version:
 +
 ----
 rpm -q smdba
 ----
 +
-PostgreSQL{nbsp}9.6 requires smdba version 1.5.8 or later.
+PostgreSQL{nbsp}10 requires ``smdba`` version 1.6.2 or later.
+
 . Perform a database backup.
 For more information on backing up, see xref:administration:backup-restore.adoc[].
 
+
+
+
+== Upgrade PostgreSQL
+
 [WARNING]
 ====
 Always create a database backup before performing a migration.
+For more information about a database backup, see xref:db-migration.adoc#db-migration-prepare[].
 ====
 
+PostgreSQL upgrades can be performed in two ways: a regular upgrade, or a fast upgrade:
 
-
-== Upgrade PostgreSQL
-
-PostgreSQL upgrades can be performed in two ways: a regular upgrade, or a fast upgrade.
-A regular upgrade will create a complete copy of the database, so you will need double the existing database size of space available.
+A *regular upgrade* will create a complete copy of the database, so you will need double the existing database size of space available.
 Regular upgrades can take a considerable amount of time, depending on the size of the database and the speed of the storage system.
 Regular upgrades also require database downtime, your database will not be accessible while the upgrade is running.
 
-A fast upgrade only takes a few minutes, and uses no additional disk space.
-However, fast upgrades are more likely to fail, and if this occurs you will need to restore the database from backup.
+A *fast upgrade* only takes a few minutes, and uses no additional disk space.
+However, fast upgrades are more likely to fail, and if this occurs you must restore the database from the backup.
```

Comment:
No, this is utterly wrong and needs to get changed! The fast upgrade is NOT "more likely to fail", in fact it is even more safe as your risk to run out of disk space is smaller. However it is more dangerous in the sense that in case of failure you do not have the implicit backup you get when doing a regular upgrade (which will copy the database files instead of hardlinking them).
This statement will scare customers and they will tend to unnecessarily waste time and space by avoiding the fast upgrade.

Reply:
thanks for clarification, Hubert.  I hope it is better worded now.